src/main/java/Main.java

```diff
@@ -68,6 +68,16 @@ public static void main(String[] args) {
         DisplayTaskTreeController displayTaskTreeController = new DisplayTaskTreeController(displayTaskTreeInteractor);
         TaskTreeUI taskTreeUI = new TaskTreeUI(windows, displayTaskTreeController);
 
+        CompleteTaskOutputBoundary completeTaskPresenter = new CompleteTaskPresenter();
+        CompleteTaskInputBoundary completeTaskInteractor = new CompleteTaskUseCase(completeTaskPresenter);
+        CompleteTaskController completeTaskController = new CompleteTaskController(completeTaskInteractor);
+
+        TaskUiOutputBoundary taskViewPresenter = new TaskUiModelPresenter(new ArrayList<>());
+        TaskUiInputBoundary taskViewInteractor = new TaskUiInteractor(taskViewPresenter);
+        TaskUiController taskViewController = new TaskUiController(taskViewInteractor);
+        TaskUI taskView = new TaskUI(windows, completeTaskController, taskViewController);
+        taskViewPresenter.addViewObserver(taskView);
```

Comment:
This seems like it's the TaskUI, rather than the actual CreateTaskUI, but considering you, David, and the rest of the group have already discussed this, it makes sense that this is what was remaining to be implemented.

Reply:
it's create "Task UI", this is the right branch